dataland-api-key-manager/src/main/kotlin/org/dataland/datalandapikeymanager/DatalandApiKeyManager.kt

```diff
@@ -1,15 +1,20 @@
 package org.dataland.datalandapikeymanager
 
 import org.dataland.datalandapikeymanager.configurations.OpenAPIConfiguration
+import org.dataland.datalandbackendutils.email.EmailSender
 import org.springframework.boot.autoconfigure.SpringBootApplication
```

Comment:
Are those requests stored somewhere?
And there is no endpoint to retrieve such requests, correct?

Reply:
Exactly. There was a discussion about that and we do not wan't a new table in this ticket. The long term plan is to have a history of requests and their handling.

Reply:
ok, thank you for clariyfing that